src/student_success_tool/gcsutils.py

```diff
@@ -0,0 +1,71 @@
+"""
+Utility functions that interact with Google Cloud Storage.
+For instance, copying files from Databricks volumes into GCP buckets.
+"""
+
+from google.cloud import storage
+from google.cloud.storage.bucket import Bucket
+from databricks.sdk.runtime import dbutils  # noqa: F401
+
+
+def save_file(
+    bucket: Bucket, src_volume_filepath: str, dest_bucket_pathname: str
+) -> None:
+    """Save file from databricks volume to a GCP bucket path."""
+    blob = bucket.blob(dest_bucket_pathname)
+    if blob.exists():
+        raise ValueError(dest_bucket_pathname + ": File already exists in bucket.")
+    blob.upload_from_filename(src_volume_filepath)
+
+
+def get_filename_from_file_path(file_path: str) -> str:
+    """Get the filename only of a given filepath, e.g. for home/directory/file.csv, return just file.csv."""
+    return file_path.split("/")[-1]
```

Comment:
iiuc, you should just use the stdlib implementation: https://docs.python.org/3/library/os.path.html#os.path.basename